_episodes/01-package-setup.md

```diff
@@ -205,11 +207,14 @@ $ cd molecool
 ### The `__init__.py` file
 
 The `__init__.py` file is a special file recognized by the Python interpreter which makes a directory into a package. This file can be blank in some cases, however, we will use it to define how the user interacts with the functions in our package.
+*TODO: Cite section on defining the interface, where we can also mention `__all__` and `_` prefixed names.*
 
+Contents of `molecool/molecool/__init__.py`:
 ~~~
 """
-molecool
-A Python package for analyzing and visualizing xyz files. For MolSSI Workshop.
+Analyze and visualize xyz files.
```

Comment:
This section will be autopopulated based on answers to the cookiecutterr prompts.

Reply:
Then it should be consistent with the generated material.

Is the docstring intentionally deviant from the PEP257 guidelines? Such as to make a point in the later lessons? If not, it seems like we should avoid mixed messages and
* put a blank line between the first and second line
* use the first line for a concise description according to PEP257 and numpy docstring conventions that will look good when formatted with `pydoc` or `help()`. This could mean just removing the `molecool\n` first line.

Reply:
@eirrgang This docstring is automatically generated by the [cookiecutter-cms](https://github.com/MolSSI/cookiecutter-cms). Therefore the above example comes automatically from the following [template](https://github.com/MolSSI/cookiecutter-cms/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.first_module_name%7D%7D.py). However, I think it is still possible to add this material on Docstring section in [Python Coding Style](https://education.molssi.org/python-package-best-practices/04-function-style/index.html).

Reply:
Thanks. Opened https://github.com/MolSSI/cookiecutter-cms/issues/130